recipes/tempo2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tempo2" %}
+{% set version = "2020.07.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  # eg, https://bitbucket.org/psrsoft/tempo2/downloads/tempo2-2019.01.1.tar.gz
+  #url: "https://bitbucket.org/psrsoft/{{ name }}/downloads/{{ name }}-{{ version }}.tar.gz"
+  ## This is v2020.07.1 but not tagged
+  url: https://bitbucket.org/psrsoft/{{ name }}/get/b054862c0788.tar.gz
+  sha256: d2d1973cc2fd660c04fc57721ab6236bf2ea197dacc9e1cc1cbe96c6cf6f27b1
+        
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - autoconf
+    - automake
+    - libtool
+  host:
+    - cfitsio
+    - pgplot
+    - gsl
+    - fftw
+    - libblas
+    - libcblas
+    - liblapack
+    - mkl
+    - llvm-openmp
+  run:
+    - cfitsio
+    - pgplot
+    - gsl
+    - fftw
+    - libblas
+    - libcblas
+    - liblapack
+    - mkl
+    - llvm-openmp
+
+test:
+  source_files:
+    - tests/test_data/test_de430.par
+    - tests/test_data/test_de430.tim
+  commands:
+    - tempo2 -f tests/test_data/test_de430.par tests/test_data/test_de430.tim
+
+about:
+  home: https://bitbucket.org/psrsoft/tempo2/
+  license: GPL-3.0-or-later
+  license_family: GPL
+  license_file: COPYING
+  summary: Tempo2 is a high precision pulsar timing tool. Tempo2 is not tempo3 either.
+  dev_url: https://bitbucket.org/psrsoft/tempo2/
+
+extra:
+  recipe-maintainers:
+    - demorest
+    - sixbynine
+    - mattpitkin
```

Comment:
Just need @sixbynine and @demorest OK to merge this.

Reply:
Looks great to me.  Thanks @mattpitkin for pushing forward with this, and @ocefpaf for the review!

Reply:
@mattpitkin we need @sixbynine OK to merge this. If you are in a hurry you can remove sixbynine for now and re-add later in the feedstock when you get the OK.

Reply:
I'm sorry that I've not been following this. I have been overloaded with teaching this semester and put all things like looking after the tempo2 repo on hold. Happy to be listed as a tempo2 maintainer for this package though.